admin/broadleaf-open-admin-platform/src/main/resources/open_admin_style/templates/layout/partials/entityTitle.html

```diff
@@ -93,17 +93,19 @@ <h4 class="main-count">
     </th:block>
 
     <th:block th:if="${entityForm}">
-        <a class="back-button" blc_admin:admin_section_href="${currentAdminSection}" >
-            <i class="fa fa-angle-left fa-2x"></i>&nbsp;&nbsp;&nbsp;
-            <h3>
-                <span th:inline="text">
-                    [[#{${currentAdminSection.sectionKey}}]]
-                </span>
-            </h3>
-        </a>
-        <h3 class="breadcrumb-divider">&nbsp;&nbsp;/&nbsp;&nbsp;</h3>
-        <h3 class="mainEntityName" th:text="${entityForm.mainEntityName}"></h3>
-        <span class="entityCatalogName" th:if="${entityCatalog != null}" th:utext="${'(' + entityCatalog.name + ')'}"></span>
+        <div th:if="${entityFriendlyName == null}">
```

Comment:
and what if entityFriendlyName is not null? I don't see usage/rendering of it...
If it suppose to be just a "flag" it should be boolean and checked as boolean